### DISCUSSION TITLE
So :execute() output no longer can be captured since 0.53.0 what's the new ideomatic way to achieve following?

### DISCUSSION THREAD

**Discussion by EnergoStalin** [junegunn/fzf#3844] [Q&A] [answered]
Your suggestions how to achieve similar behaviour to this command.

```zsh
LINK=$(
cat $TEMP | jq '.[].Name' | \
fzf --preview='cat '"$TEMP"' | jq -r ".[{n}]"' \
  --bind "enter:execute(cat '"$TEMP"' | jq -r '.[{n}].Torrent')+abort"
)
```

Would result in empty string captured and torrent url printed directly into tty as stated in changelog.

I come up with this solution using new --multi option but it's suboptimal i want to see just names in selection list having whole object as output.

```zsh
LINK=$(
cat $TEMP | jq --raw-output0 '.[]' | \
fzf \
  --read0 \
  --multi \
  --highlight-line | \
jq -r '.Torrent'
)
```

**Comment by junegunn** [comment, marked as answer]
Replace execute+abort with become.

**Reply by EnergoStalin** [reply]
Thanks it's funny but i just found this here #3155 and was about selfanswer this.
